From: Holger Assmann <h.assmann@pengutronix.de>
Date: Wed, 2 Mar 2022 09:35:15 +0100
Subject: [PATCH] ARM: stm32mp: lxa-tac: enable access to ramoops

This patch enables the ramoops-access for LXA-TAC. Regarding the
specific memory address ranges, the default values determined by Barebox
are being used; the device-tree for Linux will be patched accordingly by
Barebox.

Signed-off-by: Holger Assmann <h.assmann@pengutronix.de>
---
 arch/arm/dts/stm32mp157c-lxa-tac.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/dts/stm32mp157c-lxa-tac.dts b/arch/arm/dts/stm32mp157c-lxa-tac.dts
index 8afc610b9e6e..6301db2a337a 100644
--- a/arch/arm/dts/stm32mp157c-lxa-tac.dts
+++ b/arch/arm/dts/stm32mp157c-lxa-tac.dts
@@ -18,6 +18,13 @@
 		state = &state_emmc;
 	};
 
+	reserved-memory {
+		ramoops {
+			compatible = "ramoops";
+			/* Everything else will be fixed up at runtime */
+		};
+	};
+
 	state_emmc: state {
 		#address-cells = <1>;
 		#size-cells = <1>;
